src/StatsigOptions.ts

```diff
@@ -10,8 +13,11 @@ export default class StatsigOptions {
   public rulesUpdatedCallback: RulesUpdatedCallback | null;
   public localMode: boolean;
   public initTimeoutMs: number;
+  public configAdapter: IConfigAdapter | null;
+  public loggingAdapter: ILoggingAdapter | null;
+  public idListAdapter: IIDListAdapter | null;
```

Comment:
So this is a bit of a pain - now I have to implement 3 different adapters.  Instead, just make a single storage adapter, and we will handle writing a different key/value pair for configs/logs/idlists/etc

Reply:
Good point. I was thinking this would give more flexibility in which adapters to apply but we can achieve that by having an in-memory default adapter